app/scripts/services/Session.svc.js

```diff
@@ -24,11 +26,33 @@ angular.module('yourStlCourts').factory('Session', function () {
     lastSelectedCitation = null;
   }
 
+  function setLatestDOB(dob){
+    latestDOB = formatDOB(dob);
+  }
+
+  function getLatestDOB(){
+    return latestDOB;
+  }
+
+  function formatDOB(dob){
```

Comment:
What we really need to do is store an actual JavaScript `Date` object and then format using the angular filters upon display in the template: https://docs.angularjs.org/api/ng/filter/date. Please convert your `yyyy-mm-dd` to `Date`